### PR TITLE
#11731 Release 22.10.0

### LINKED ISSUE #11731: Release 22.10.0
This is the ticket to track the release of 22.10.0

### PR DESCRIPTION
**Pull request by adiroiban**
## Scope and purpose

Fixes #11731

This is the release branch fro 22.10.0

It was done to handle the low-security issue in twisted web.

## Links

* pypi - https://pypi.org/project/Twisted/22.10.0rc1/
* RC1 - GitHub Release https://github.com/twisted/twisted/releases/tag/twisted-22.10.0rc1
* RC1 - GitHub Actions on tag https://github.com/twisted/twisted/actions/runs/3331884097/jobs/5512138161

### CODE REVIEW

**Comment by adiroiban** [comment]
I think that this is ready for a first review.

so, please review. Thanks!

File: NEWS.rst [hunk 1]
```diff
@@ -8,6 +8,120 @@ https://twisted.org/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.10.0.rc1 (2022-10-26)
+================================
+
+This release contains a security fix for CVE-2022-39348.
+This is a low security bug.
```
Comment (twm) [inline NEWS.rst]:
```suggestion
This is a low severity security bug.
```

File: NEWS.rst [hunk 2]
```diff
@@ -8,6 +8,120 @@ https://twisted.org/trac/ticket/<number>
...
+Features
+--------
+
+- The ``systemd:`` endpoint parser now supports "named" file descriptors.  This is a more reliable mechanism for choosing among several inherited descriptors. (#8147)
+
+
+Bugfixes
+--------
+
+- `test.yaml` workflow permissions restricted. (#11631)
```
Comment (twm) [inline NEWS.rst]: This has no user-facing impact, so it should be in Misc (see suggestion below).

```suggestion
```

File: NEWS.rst [hunk 3]
```diff
@@ -8,6 +8,120 @@ https://twisted.org/trac/ticket/<number>
...
+----------------------
+
+- The ``systemd`` endpoint parser's ``index`` parameter is now documented as leading to non-deterministic results in which descriptor is selected.  The new ``name`` parameter is now documented as preferred. (#8146)
+- The implementers of Zope interfaces are once more displayed in the documentations. (#11690)
+
+
+Deprecations and Removals
+-------------------------
+
+- twisted.protocol.dict, which was deprecated in 17.9, was now removed. (#11725)
```
Comment (twm) [inline NEWS.rst]:
```suggestion
- twisted.protocols.dict, which was deprecated in 17.9, has been removed. (#11725)
```

File: NEWS.rst [hunk 4]
```diff
@@ -8,6 +8,120 @@ https://twisted.org/trac/ticket/<number>
...
+
+
+Misc
+----
+
+- #11573, #11599, #11616, #11628, #11640, #11645, #11647, #11652, #11664, #11674, #11679, #11686, #11692, #11694, #11696, #11700, #11702, #11713, #11715, #11721
```
Comment (twm) [inline NEWS.rst]: Add the ticket number from above.

```suggestion
- #11573, #11599, #11616, #11628, #11631, #11640, #11645, #11647, #11652, #11664, #11674, #11679, #11686, #11692, #11694, #11696, #11700, #11702, #11713, #11715, #11721
```

**Review by twm — APPROVED** [review]
I suggested a few changes to the release notes, but overall, looks good. Thank you for managing the release @adiroiban!

**Comment by adiroiban** [comment]
22.10.0 publish action here  https://github.com/twisted/twisted/actions/runs/3360096602/jobs/5568877037

**Pull request merged**